src/Giraffe/HttpHandlers.fs

```diff
@@ -447,4 +447,7 @@ let negotiateWith (rules : IDictionary<string, obj -> HttpHandler>) (responseObj
             <| ctx
 
 let negotiate (responseObj : obj) =
-    negotiateWith defaultNegotioationRules responseObj
\ No newline at end of file
+    negotiateWith defaultNegotioationRules responseObj
+
+let warbler f a = 
```

Comment:
Indeed, looks like [suave](https://github.com/SuaveIO/suave/blob/b8d098f0142db23806d44a7e5555606ccaaa89dd/src/Suave/WebPart.fs#L59) did this as well. I'll change it.